### PR TITLE
update sonarr v4 install docs && servarr script

### PR DESCRIPTION
**Pull request by bakerboy448**

### CODE REVIEW

File: install-script.md [hunk 1]
```diff
@@ -34,269 +34,14 @@ Be aware of the following:
 
 > SSH in using Putty, mRemoteNG, or any other SSH tool. Note that most tools support saving your connection.{.is-info}
 
-- Once SSHed in type the below to create the installation script in your current directory
+- Once SSHed in type the below to download the installation script in your current directory
 
 ```bash
-nano ArrInstall.sh
+curl -o starrInstall.sh https://raw.githubusercontent.com/Servarr/Wiki/master/servarr/servarr-install-script.sh
```
Comment (stevietv) [inline install-script.md:40]:
```suggestion
curl -o servarr-install-script.sh https://raw.githubusercontent.com/Servarr/Wiki/master/servarr/servarr-install-script.sh
```

Why change the name on download? starr doesn't mean so much when its referred to as Servarr Installation Script

File: install-script.md [hunk 2]
```diff
@@ -34,269 +34,14 @@ Be aware of the following:
...
-- Press <kbd>Ctrl</kbd>+<kbd>O</kbd> (save) then <kbd>Enter</kbd>
-- Press <kbd>Ctrl</kbd>+<kbd>X</kbd> (exit) then <kbd>Enter</kbd>
-- Then in your console type and follow the prompts
+- To run the install script
 
 ```shell
-sudo bash ArrInstall.sh
-```
\ No newline at end of file
+sudo bash starrInstall.sh
```
Comment (stevietv) [inline install-script.md:46]:
```suggestion
sudo bash servarr-install-script.sh
```

in line with above.
didn't you want to make it a piped install?

File: servarr/servarr-install-script.sh [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+#LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+#OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+#WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+scriptversion="3.0.10"
+scriptdate="2024-01-04"
+
+set -euo pipefail
+
+echo "Running \*Arr Install Script - Version [$scriptversion] as of [$scriptdate]"
```
Comment (stevietv) [inline servarr/servarr-install-script.sh:36]:
```suggestion
echo "Running Servarr Install Script - Version [$scriptversion] as of [$scriptdate]"
```

File: servarr/servarr-install-script.sh [hunk 4]
```diff
@@ -0,0 +1,241 @@
+#!/bin/bash
+### Description: \*Arr .NET Debian install
```
Comment (stevietv) [inline servarr/servarr-install-script.sh:2]:
```suggestion
### Description: Servarr .NET Debian install
```

**Review by stevietv — CHANGES_REQUESTED** [review]

File: install-script.md [hunk 5]
```diff
@@ -34,269 +34,14 @@ Be aware of the following:
 
 > SSH in using Putty, mRemoteNG, or any other SSH tool. Note that most tools support saving your connection.{.is-info}
 
-- Once SSHed in type the below to create the installation script in your current directory
+- Once SSHed in type the below to download the installation script in your current directory
```
Comment (zakkarry) [inline install-script.md:37]:
```suggestion
- Once SSHed in type the command below to download the installation script in your current directory
```

File: radarr/installation/macos.md [hunk 6]
```diff
@@ -18,7 +18,8 @@ dateCreated: 2023-07-03T20:12:05.036Z
 1. Download the [MacOS App](https://radarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=x64&installer=true) or  the [MacOS M1 App](https://radarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=arm64&installer=true) depending on your system
```
Comment (zakkarry) [inline radarr/installation/macos.md:18]:
```suggestion
1. Download the [MacOS App](https://radarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=x64&installer=true) or  the [MacOS M1 App](https://radarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=arm64&installer=true) depending on your system architecture.
```

File: prowlarr/installation/macos.md [hunk 7]
```diff
@@ -18,4 +18,5 @@ dateCreated: 2023-07-03T20:11:30.382Z
 1. Download the [MacOS App](https://prowlarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=x64&installer=true) or  the [MacOS M1 App](https://prowlarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=arm64&installer=true) depending on your system
```
Comment (zakkarry) [inline prowlarr/installation/macos.md:18]:
```suggestion
1. Download the [MacOS App](https://prowlarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=x64&installer=true) or  the [MacOS M1 App](https://prowlarr.servarr.com/v1/update/master/updatefile?os=osx&runtime=netcore&arch=arm64&installer=true) depending on your system architecture.
```

**Review by zakkarry — COMMENTED** [review]
Just a few nits that I would put personally, take it or leave it :P

**Pull request merged**